packages/server/postgres/migrations/1658834062488_createMeetingSeries.ts

```diff
@@ -0,0 +1,33 @@
+import {Client} from 'pg'
+import getPgConfig from '../getPgConfig'
+
+export async function up() {
+  const client = new Client(getPgConfig())
+  await client.connect()
+  await client.query(`
+    CREATE TABLE IF NOT EXISTS "MeetingSeries" (
+      "id" INT GENERATED BY DEFAULT AS IDENTITY PRIMARY KEY,
+      "meetingType" "MeetingTypeEnum" NOT NULL,
+      "title" TEXT NOT NULL,
+      "recurrenceRule" TEXT NOT NULL,
+      "duration" INT NOT NULL,
+      "createdAt" TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP NOT NULL,
+      "updatedAt" TIMESTAMP WITH TIME ZONE DEFAULT CURRENT_TIMESTAMP NOT NULL,
+      "cancelledAt" TIMESTAMP WITH TIME ZONE
+    );
+
+    DROP TRIGGER IF EXISTS "update_MeetingSeries_updatedAt" ON "MeetingSeries";
```

Comment:
+1 do we actually need to drop it here or is it just to be safe?

Reply:
Just to be safe - similar to a `CREATE TABLE IF NOT EXISTS` but for triggers